--- a/src/ispyb/models/_auto_db_schema.py	2022-07-18 09:48:17.000000000 +0200
+++ b/src/ispyb/models/_auto_db_schema_patched.py	2022-07-18 10:21:46.000000000 +0200
@@ -27,8 +27,10 @@
 from sqlalchemy.orm import relationship
 from sqlalchemy.ext.declarative import declarative_base
 
+from .base import CustomBase
 
-Base = declarative_base()
+
+Base = declarative_base(cls=CustomBase)
 metadata = Base.metadata
 
 
@@ -2706,15 +2707,15 @@
     comments = Column(String(512))
     creationTime = Column(DateTime)
 
-    Model = relationship(
+    AveragedModel = relationship(
         "Model", primaryjoin="AbInitioModel.averagedModelId == Model.modelId"
     )
     ModelList = relationship("ModelList")
-    Model1 = relationship(
+    RapidShapeDeterminationModel = relationship(
         "Model",
         primaryjoin="AbInitioModel.rapidShapeDeterminationModelId == Model.modelId",
     )
-    Model2 = relationship(
+    ShapeDeterminationModel = relationship(
         "Model", primaryjoin="AbInitioModel.shapeDeterminationModelId == Model.modelId"
     )
 
@@ -5878,10 +5879,6 @@
         "DataCollection",
         primaryjoin="Image.dataCollectionId == DataCollection.dataCollectionId",
     )
-    DataCollection1 = relationship(
-        "DataCollection",
-        primaryjoin="Image.dataCollectionId == DataCollection.dataCollectionId",
-    )
 
 
 class Movie(Base):
